apps/artemis-web-e2e/src/integration/app2.spec.ts

```diff
@@ -1,16 +1,10 @@
-import Chance from 'chance';
-
-let newEmail;
-let newPass;
-
 describe('artemis-web', () => {
   beforeEach(() => {
     Cypress.Cookies.preserveOnce('remember_me', 'sid', 'access_token');
   });
 
   it('logs in with ldap', () => {
     cy.visit('/login');
-    cy.wait(2000);
```

Comment:
btw why do we have app and app2? Why are they named this way?

Reply:
to avoid memory errors in the cypress action I tried to split the files

Reply:
OK I see (as long as the prod is clean, OK). Resolving.